### ISSUE TITLE
Alpha installation fails in conda environment

### ISSUE THREAD

**Issue by asifm** [explosion/spaCy#1247]
Couldn't install after creating a new environment with conda and then running pip install -U spacy-nightly. Console log here: https://www.dropbox.com/s/ut6b6ef4jmy2e2r/spacy-nightly-installation-failed.html?dl=0

## Your Environment

* Operating System: Windows 10
* Python Version Used: 3.6.2
* spaCy Version Used: alpha
* Environment Information: conda separate environment

**Comment by ines** [comment]
Thanks for the report! Looks like this is related to Thinc... I'm copying over the most relevant part for future reference, and we'll look into this!

```
  Failed building wheel for thinc
  Running setup.py clean for thinc
Failed to build spacy-nightly thinc
Installing collected packages: thinc, ujson, chardet, urllib3, idna, requests, regex, wcwidth, webencodings, setuptools, html5lib, ftfy, spacy-nightly
  Running setup.py install for thinc ... error
    Complete output from command C:\Users\xxx\AppData\Local\Continuum\Anaconda3\envs\spacy-nightly\python.exe -u -c &quot;import setuptools, tokenize;__file__=&#39;C:\\Users\\xxx\\AppData\\Local\\Temp\\pip-build-1bp4_sk9\\thinc\\setup.py&#39;;f=getattr(tokenize, &#39;open&#39;, open)(__file__);code=f.read().replace(&#39;\r\n&#39;, &#39;\n&#39;);f.close();exec(compile(code, __file__, &#39;exec&#39;))&quot; install --record C:\Users\xxx\AppData\Local\Temp\pip-3t0if_eo-record\install-record.txt --single-version-externally-managed --compile:
    Warning: The nvcc binary could not be located in your $PATH. For GPU capability, either add it to your path, or set $CUDA_HOME
    running install
    running build
    running build_py
    running build_ext
    building &#39;thinc.linalg&#39; extension
    C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\BIN\x86_amd64\cl.exe /c /nologo /Ox /W3 /GL /DNDEBUG /MD -IC:\Users\xxx\AppData\Local\Continuum\Anaconda3\envs\spacy-nightly\include -IC:\Users\xxx\AppData\Local\Temp\pip-build-1bp4_sk9\thinc\include -IC:\Users\xxx\AppData\Local\Continuum\Anaconda3\envs\spacy-nightly\include -IC:\Users\xxx\AppData\Local\Continuum\Anaconda3\envs\spacy-nightly\include &quot;-IC:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\INCLUDE&quot; &quot;-IC:\Program Files (x86)\Windows Kits\10\include\10.0.10240.0\ucrt&quot; &quot;-IC:\Program Files (x86)\Windows Kits\NETFXSDK\4.6.1\include\um&quot; &quot;-IC:\Program Files (x86)\Windows Kits\8.1\include\shared&quot; &quot;-IC:\Program Files (x86)\Windows Kits\8.1\include\um&quot; &quot;-IC:\Program Files (x86)\Windows Kits\8.1\include\winrt&quot; /EHsc /Tpthinc/linalg.cpp /Fobuild\temp.win-amd64-3.6\Release\thinc/linalg.obj gcc nvcc
    cl : Command line warning D9024 : unrecognized source file type &#39;gcc&#39;, object file assumed
    cl : Command line warning D9027 : source file &#39;gcc&#39; ignored
    cl : Command line warning D9024 : unrecognized source file type &#39;nvcc&#39;, object file assumed
    cl : Command line warning D9027 : source file &#39;nvcc&#39; ignored
    linalg.cpp
    c1xx: fatal error C1083: Cannot open source file: &#39;thinc/linalg.cpp&#39;: No such file or directory
    error: command &#39;C:\\Program Files (x86)\\Microsoft Visual Studio 14.0\\VC\\BIN\\x86_amd64\\cl.exe&#39; failed with exit status 2
```

**Comment by CYUlysses** [comment]
Hi, I encountered the same problem. I tried running from both the root Anaconda python 3.6.1 and in a virtual environment with Python 3.6.2, and encountered the same problem.

relevant error codes below:

`error: command 'C:\\Program Files (x86)\\Microsoft Visual Studio 14.0\\VC\\BIN\\x86_amd64\\cl.exe' failed with exit status 2`

And after this error:
`  Failed building wheel for thinc
  Running setup.py clean for thinc
Failed to build spacy-nightly thinc
Installing collected packages: thinc, ujson, regex, ftfy, spacy-nightly
  Running setup.py install for thinc ... error
    Complete output from command C:\Users\cydru\Anaconda3\python.exe -u -c "import setuptools, tokenize;__file__='C:\\Users\\cydru\\AppData\\Local\\Temp\\pip-build-p63_q889\\thinc\\setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" install --record C:\Users\cydru\AppData\Local\Temp\pip-j5xhl_x5-record\install-record.txt --single-version-externally-managed --compile:
    Warning: The nvcc binary could not be located in your $PATH. For GPU capability, either add it to your path, or set $CUDA_HOME
    running install
    running build
    running build_py
    running build_ext
    building 'thinc.linalg' extension
    C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\BIN\x86_amd64\cl.exe /c /nologo /Ox /W3 /GL /DNDEBUG /MD -IC:\Users\cydru\Anaconda3\include -IC:\Users\cydru\AppData\Local\Temp\pip-build-p63_q889\thinc\include -IC:\Users\cydru\Anaconda3\include -IC:\Users\cydru\Anaconda3\include "-IC:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\INCLUDE" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.10240.0\ucrt" "-IC:\Program Files (x86)\Windows Kits\8.1\include\shared" "-IC:\Program Files (x86)\Windows Kits\8.1\include\um" "-IC:\Program Files (x86)\Windows Kits\8.1\include\winrt" /EHsc /Tpthinc/linalg.cpp /Fobuild\temp.win-amd64-3.6\Release\thinc/linalg.obj gcc nvcc
    cl : Command line warning D9024 : unrecognized source file type 'gcc', object file assumed
    cl : Command line warning D9027 : source file 'gcc' ignored
    cl : Command line warning D9024 : unrecognized source file type 'nvcc', object file assumed
    cl : Command line warning D9027 : source file 'nvcc' ignored
    linalg.cpp
    c1xx: fatal error C1083: Cannot open source file: 'thinc/linalg.cpp': No such file or directory
    error: command 'C:\\Program Files (x86)\\Microsoft Visual Studio 14.0\\VC\\BIN\\x86_amd64\\cl.exe' failed with exit status 2`

**Comment by alenboby** [comment]
Hai, any update here?
i got the same error and still have no clue

**Comment by ines** [comment]
This should be fixed in the new version, v2.0. We're currently waiting to get it built for conda (it's just taking a bit longer because of Travis errors and the OSX backlogs 😓). 

Once the version is up, could you try again? [See here](https://spacy.io/usage/v2) for details on what's new in v2.0. If there are still problems with conda install and the new version, feel free to reopen.

Thinc 6.10 is already up on `conda-forge` btw!

**Comment by dineshdharme** [comment]
I have the same error. Here is the issue I raised on spacy : https://github.com/explosion/spaCy/issues/1745
I have copied the details here.


So I was following the guide to install spacy with gpu support : https://spacy.io/usage/
I already have cuda installed with CUDA_HOME variable set as mentioned in the above link 
I ran the following command. I have both python2.7 and python3 (-> python3.6)

```
sudo pip3 install spacy
python3 -c "import thinc.neural.gpu_ops"  

```
I got the following error,

Traceback (most recent call last):
  File "<string>", line 1, in <module>
ModuleNotFoundError: No module named 'thinc.neural.gpu_ops'

I have successfully installed cupy from Chainer as well. Then uninstalled thinc and again installed thinc. But same error.

I uninstalled thinc and then installed it again with specifying the install log file.
sudo pip3 install thinc --log thinclogs.txt --no-cache-dir

From the logs, it seems that it is unable to find nvcc. But I can clearly access nvcc with nvcc -V command on the terminal.
Here are the relevant logs. 

Downloading thinc-6.10.2.tar.gz (1.2MB)
  Downloading from URL https://pypi.python.org/packages/55/fd/e9f36081e6f53699943381858848f3b4d759e0dd03c43b98807dde34c252/thinc-6.10.2.tar.gz#md5=41f4cd26d3e894600cdd36e3cd074239 (from https://pypi.python.org/simple/thinc/)
  Running setup.py (path:/tmp/pip-build-xmgkus0e/thinc/setup.py) egg_info for package thinc
    Running command python setup.py egg_info
    Warning: The nvcc binary could not be located in your $PATH. For GPU capability, either add it to your path, or set $CUDA_HOME
    running egg_info
    creating pip-egg-info/thinc.egg-info
    writing pip-egg-info/thinc.egg-info/PKG-INFO
    writing dependency_links to pip-egg-info/thinc.egg-info/dependency_links.txt
    writing requirements to pip-egg-info/thinc.egg-info/requires.txt
    writing top-level names to pip-egg-info/thinc.egg-info/top_level.txt
    writing manifest file 'pip-egg-info/thinc.egg-info/SOURCES.txt'
    reading manifest file 'pip-egg-info/thinc.egg-info/SOURCES.txt'
    reading manifest template 'MANIFEST.in'
    warning: no files found matching '*.cpp' under directory 'include'
    warning: no files found matching 'buildbot.json'
    writing manifest file 'pip-egg-info/thinc.egg-info/SOURCES.txt'


## Your Environment
* **spaCy version:** 2.0.5
* **Platform:** Linux-4.13.0-19-generic-x86_64-with-Ubuntu-17.10-artful
* **Python version:** 3.6.3
* Environment Information: I have Cuda 8 with gtx 1080
